pkg/distro/fedora/distro.go

```diff
@@ -124,7 +124,7 @@ var (
 		packageSets: map[string]packageSetFunc{
 			osPkgsKey: iotCommitPackageSet,
 			containerPkgsKey: func(t *imageType) rpmmd.PackageSet {
-				return rpmmd.PackageSet{}
+				return rpmmd.PackageSet{InstallWeakDeps: false}
```

Comment:
nitpick: you don't need to set the option to `false`, since it is the default value for the type...

Reply:
Done :)